README.md

```diff
@@ -63,7 +63,20 @@ conn:close() -- Close stmt as well.
 
 ## Install
 
-- Install the lit package:
+1. Install sqlite:
+Linux: `sudo apt-get install sqlite3 libsqlite3-dev`
+Linux (build from source):
+```
+sudo wget https://www.sqlite.org/2022/sqlite-autoconf-3380500.tar.gz
```

Comment:
We can't realistically maintain this URL, since it changes regularly.

Reply:
The url contains the sqlite version.
yes, we cant maintain url to the current version - but I think its not necessary, its shouldnt be a problem if you use a program that was released last year or earlier.
ask debian users if it really matters if the software is a little outdated?